.github/workflows/r_nightly.yml

```diff
@@ -65,54 +68,58 @@ jobs:
           fi
           echo $PREFIX
 
-          archery crossbow download-artifacts -f r-nightly-packages -t binaries --skip-pattern-validation $PREFIX
+          archery crossbow download-artifacts -f r-nightly-packages -t binaries $PREFIX
+
+          if [ -n "$(ls -A binaries/*/*/)" ]; then
+            echo "Found files!"
+            echo "GOT_FILES=true" >> $GITHUB_ENV
+          else
+            echo "No files found. Stopping upload."

```

Comment:
Why not just `exit 1` and fail the workflow if there are no artifacts to upload? Seems like there is a risk that packages will silently fail to upload. (It's a risk anyway because no one seems to be alerted when cron workflows fail.)

Reply:
If there are no files it is most likely that the crossbow job failed (which would be in the emails/report messages), but thinking about it, it might be best to only have this job green if the upload was successful and red otherwise. 

We could also add a step/job where we test install from the repo and check for the correct version? This was part of arrow-r-nightly but is gone now because of the local repo system in the crossbow job...

Reply:
Yeah I thought about that too, might be useful to confirm that the repo structure we're uploading is valid

Reply:
I'll make a note and probably add it when I introduce the pruning.